packages/applications-service-api/src/utils/representation.mapper.js

```diff
@@ -43,14 +43,24 @@ const mapCommonRepresentationBOFieldsToApi = (representation) => {
 		Representative,
 		OrgOnBhalfName: represented.organisationName,
 		RepFrom: representation.representationType,
+		repFromWelsh: repFromWelshDictionary[representation.representationType],
 		RepresentationRedacted: representation.representationComment,
 		DateRrepReceived: representation.dateReceived,
 		Attachments: representation.attachmentIds
 	};
 };
 
+const repFromWelshDictionary = {
```

Comment:
Do we want to handle capitalisation?  Just so we cover off the inevitable lookup for 'Local authorities'.